pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -44,6 +44,7 @@ func publishContentTreeStatus(ctx *volumemgrContext, status *types.ContentTreeSt
 	log.Debugf("publishContentTreeStatus(%s)", key)
 	pub := ctx.pubContentTreeStatus
 	pub.Publish(key, *status)
+	updateVolumeStatusFromContentID(ctx, status.ContentID)
```

Comment:
Why do we need to bury the need to update as a side effect of a publish function?

Reply:
Corrected.